app/templates/search.html

```diff
@@ -95,6 +149,7 @@
 {% endblock %}
 
 {% block scripts %}
-  <script src="{{ static('js/navbar.js') }}"></script>
+  <script src="{{ static('js/search.js') }}"></script>
+  <script src="{{ static('js/navBar.js') }}"></script>
```

Comment:
Wrong merge choice here. Should be just `<script src="{{ static('js/navbar.js') }}"></script>`. Note the lower case b as well.